### PR TITLE
Refactor errors naming to follow convention

### PR DESCRIPTION
**Pull request by Psirex**

### CODE REVIEW

File: test/unit/Escrow.t.sol [hunk 1]
```diff
@@ -106,12 +106,12 @@ contract EscrowUnitTests is UnitTest {
     // ---
 
     function test_getUnclaimedUnstETHIdsCount_RevertOn_UnexpectedState_Signaling() external {
```
Comment (sandstone-ag) [inline test/unit/Escrow.t.sol:108]:
```suggestion
    function test_getUnclaimedUnstETHIdsCount_RevertOn_UnexpectedEscrowState_Signaling() external {
```

File: test/unit/Escrow.t.sol [hunk 2]
```diff
@@ -106,12 +106,12 @@ contract EscrowUnitTests is UnitTest {
...
-        vm.expectRevert(abi.encodeWithSelector(EscrowState.UnexpectedState.selector, State.RageQuitEscrow));
+        vm.expectRevert(abi.encodeWithSelector(EscrowState.UnexpectedEscrowState.selector, State.SignallingEscrow));
         _escrow.getUnclaimedUnstETHIdsCount();
     }
 
     function test_getUnclaimedUnstETHIdsCount_RevertOn_UnexpectedState_NotInitialized() external {
```
Comment (sandstone-ag) [inline test/unit/Escrow.t.sol:113]:
```suggestion
    function test_getUnclaimedUnstETHIdsCount_RevertOn_UnexpectedEscrowState_NotInitialized() external {
```

File: test/unit/Escrow.t.sol [hunk 3]
```diff
@@ -121,13 +121,13 @@ contract EscrowUnitTests is UnitTest {
 
     function test_getNextWithdrawalBatch_RevertOn_UnexpectedState_Signaling() external {
```
Comment (sandstone-ag) [inline test/unit/Escrow.t.sol:122]:
```suggestion
    function test_getNextWithdrawalBatch_RevertOn_UnexpectedEscrowState_Signaling() external {
```

File: test/unit/Escrow.t.sol [hunk 4]
```diff
@@ -121,13 +121,13 @@ contract EscrowUnitTests is UnitTest {
...
         uint256 batchLimit = 10;
-        vm.expectRevert(abi.encodeWithSelector(EscrowState.UnexpectedState.selector, State.RageQuitEscrow));
+        vm.expectRevert(abi.encodeWithSelector(EscrowState.UnexpectedEscrowState.selector, State.SignallingEscrow));
         _escrow.getNextWithdrawalBatch(batchLimit);
     }
 
     function test_getNextWithdrawalBatch_RevertOn_UnexpectedState_NotInitialized() external {
```
Comment (sandstone-ag) [inline test/unit/Escrow.t.sol:128]:
```suggestion
    function test_getNextWithdrawalBatch_RevertOn_UnexpectedEscrowState_NotInitialized() external {
```

File: test/unit/Escrow.t.sol [hunk 5]
```diff
@@ -136,12 +136,12 @@ contract EscrowUnitTests is UnitTest {
     // ---
 
     function test_isWithdrawalsBatchesClosed_RevertOn_UnexpectedState_Signaling() external {
```
Comment (sandstone-ag) [inline test/unit/Escrow.t.sol:138]:
```suggestion
    function test_isWithdrawalsBatchesClosed_RevertOn_UnexpectedEscrowState_Signaling() external {
```

File: test/unit/Escrow.t.sol [hunk 6]
```diff
@@ -136,12 +136,12 @@ contract EscrowUnitTests is UnitTest {
...
-        vm.expectRevert(abi.encodeWithSelector(EscrowState.UnexpectedState.selector, State.RageQuitEscrow));
+        vm.expectRevert(abi.encodeWithSelector(EscrowState.UnexpectedEscrowState.selector, State.SignallingEscrow));
         _escrow.isWithdrawalsBatchesClosed();
     }
 
     function test_isWithdrawalsBatchesClosed_RevertOn_UnexpectedState_NotInitialized() external {
```
Comment (sandstone-ag) [inline test/unit/Escrow.t.sol:143]:
```suggestion
    function test_isWithdrawalsBatchesClosed_RevertOn_UnexpectedEscrowState_NotInitialized() external {
```

**Review by sandstone-ag — APPROVED** [review]
LGTM, but consider changing test names for consistency.

**Review by bulbozaur — APPROVED** [review]

**Pull request merged**